birdy/client/base.py

```diff
@@ -112,6 +96,40 @@ def __init__(
 
         self.__doc__ = utils.build_wps_client_doc(self._wps, self._processes)
 
+    def _get_process_description(self, processes):
+        """Return the description for each process.
+
+        Sends the server a `describeProcess` request for each process.
+
+        Parameters
+        ----------
+        processes: str, list, None
+          A process name, a list of process names or None (for all processes).
+
+        Returns
+        -------
+        OrderedDict
+          A dictionary keyed by the process identifier of process descriptions.
+        """
+        if processes is None:
+            # Get the description for all processes in one request.
+            ps = self._wps.describeprocess('all')
```

Comment:
In this case we assume, we always get a list of processes. Which is right :) But need to adapt owslib PR.

Reply:
How about backward-compatibility? I can also make a pre-release package of owslib on the birdhouse channel.

Reply:
Your're right, if a server had only one process, this would break. 

I guess we should try to do something backward compatible, but doing this elegantly might be complicated.

Reply:
Last push includes a fix to make the change backward compatible. It only requires owslib to bump version. The change almost divides the test run time by half.